### DISCUSSION TITLE
AdGuard for Windows

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#87] [General]
# AdGuard for Windows

Check if AdGuard for Windows is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/64eb4c91-d336-4682-bf88-b35c8b4071bd

**Comment by Marcelino-Verazani** [comment]
AdGuard for Windows already runs native in WoA since 7.11 released at 11/17/22: 
https://adguard.com/en/blog/adguard-v7-11-for-windows.html

**Reply by viveksh1** [reply]
It supports WoA with a WoA driver, but I am not sure of the AdGuard program itself runts natively, or just in emulation mode with a native driver.

**Reply by OpenSource03** [reply]
For status changes please click on report an issue button :)
